src/test/kotlin/sirius/web/http/CSRFTokenTest.kt

```diff
@@ -0,0 +1,181 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.web.http
+
+import io.netty.handler.codec.http.HttpHeaderNames
+import io.netty.handler.codec.http.HttpResponseStatus
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import sirius.kernel.SiriusExtension
+import sirius.kernel.commons.Streams
+import java.net.HttpURLConnection
+import java.net.URL
+
+import java.nio.charset.StandardCharsets
+import kotlin.test.assertEquals
+
+@ExtendWith(SiriusExtension::class)
+class CSRFTokenTest {
+
+    @Test
+    fun `safePOST() works correctly if token is missing via GET`() {
```

Comment:
Maybe add more meaningful description than "works correctly" :-)
E.g.: "safePOST() fails if token is missing via GET" or "safePOST() fails on GET with missing token"